pyigm/clustering/xcorr_utils.py

```diff
@@ -41,6 +48,12 @@ def auto_pairs_rt(X, Y, Z, rbinedges, tbinedges, wrap=True, track_time=False):
         # transverse separation
         tsep = np.hypot(y[i+1:] - y[i], z[i+1:] - z[i])
 
+        # Cut me (3x speed-up)
+        ok_sep = (rsep <= rbinedges[-1]) & (tsep <= tbinedges[-1])
```

Comment:
In the case of `wrap` is `False`, then an extra condition should be set to eliminate those that are smaller than the minimum (negative) rbinedges value

Reply:
added